server/graphql/general/src/lib.rs

```diff
@@ -437,6 +440,15 @@ impl GeneralMutations {
     ) -> Result<UpdateLabelPrinterSettingsResponse> {
         update_label_printer_settings(ctx, input)
     }
+
+    pub async fn update_name_properties(
```

Comment:
What do you imagine we'd do differently one each? I think this is fine from what I understand...

Reply:
This was considering a restriction of what you could edit via remote site vs central, but that would require more of a refactor anyway, because we currently just submit the new stringified properties 🤷‍♀️